bigquery/deploy/README.md

```diff
@@ -47,6 +47,8 @@
     * Go to https://console.cloud.google.com/kubernetes/list and click `Create Cluster`
     * Change name to `elasticsearch-cluster`
     * Change `Machine type` to `4 vCPUs`. (Otherwise will get Insufficient CPU error.)
+    If you need more memory for Elasticsearch, you may need `n1-highmem-4`. [Elasticsearch recommends](https://www.elastic.co/guide/en/elasticsearch/reference/current/heap-size.html) the VM has at least
```

Comment:
Change data-explorer-indexers/tree/master/kubernetes-elasticsearch-cluster to say

The configs here were taken from the kubernetes-elasticsearch-cluster repository with some modifications:
- Using [StatefulSet](https://github.com/pires/kubernetes-elasticsearch-cluster/tree/master/stateful) for persistent storage
- Be able to support large datasets/indices
- Internal load balancing

Reply:
Done.